src/ploneintranet/workspace/browser/add_content.py

```diff
@@ -92,10 +96,25 @@ def create(self):
                     participant_policy = 'publishers'
                 else:
                     raise AttributeError
-
+            old_security_manager = getSecurityManager()
+            acl_users = api.portal.get_tool('acl_users')
+            tmp_user = UnrestrictedUser(
+                'Workspace Init', '', ['TeamManager'], '')
+            tmp_acl_user = tmp_user.__of__(acl_users)
+            newSecurityManager(None, tmp_acl_user)
+            try:
                 new.set_external_visibility(external_visibility)
```

Comment:
File "/var/lib/jenkins/workspace/Plone.Intranet.Pull.Request.Builder/src/ploneintranet/workspace/browser/add_content.py", line 106, in create
    new.set_external_visibility(external_visibility)
UnboundLocalError: local variable 'external_visibility' referenced before assignment

This is because upwards (not shown in the diff) there's no fallback clause to define this var if there is no 'form'.